### ISSUE TITLE
Why not add something like `FB_LOG_EVERY_MS` into glog?

### ISSUE THREAD

**Issue by GOGOYAO** [google/glog#884]
Somtimes, it is more usefull to log every n ms. It is available in folly, why not support it in glog.

**Comment by Mivekk** [comment]
It is supported in glog!
You should use `LOG_EVERY_T`

Here is an example with std::chrono_literals:
```
using namespace std::chrono_literals;
LOG_EVERY_T(INFO, 123ms) << "Hello!";
```

If you don't want to use std::chrono_literals you can express duration as a fraction of a second:
`LOG_EVERY_T(INFO, 0.123) << "Hello!";`

**Comment by sergiud** [comment]
@Mivekk Thanks. 

Indeed, this feature has been already implemented in #669 and is available since glog 0.6.0.
